test/cpp/qps/client_async.cc

```diff
@@ -316,23 +317,28 @@ class AsyncClient : public Client {
   }
 
  private:
-  class boolean { // exists only to avoid data-race on vector<bool>
+  class boolean {  // exists only to avoid data-race on vector<bool>
```

Comment:
It'd be interested to know the back-story on this one... I'm aware of the template specialization that happens in the STL for vector<bool>, but which data-races was it causing?